app/public/stylesheets/style.scss

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   Base
+   ========================================================================== */
+
+/**
+ * Base values
+ */
+
+$disclaimer-color: #6FC9C0;
+$headline-color: #3F5EAE;
+$button-font-color: #fff;
+
+div,
+input,
+p {
+  font-family: 'Press Start 2P', cursive;
```

Comment:
I would assign this rule to body since everything uses the same font. Another benefit of doing that is to remove the tag selectors.